src/modules/legacy/components/AccountsComparison/index.js

```diff
@@ -77,47 +75,45 @@ const AccountsComparison = ({ t }) => {
                 </>
               </Tooltip>
               <br />
-              {!hasEnoughBalance && (
-                <>
-                  <span>
-                    {t('An initial one-time transfer fee will be deducted from the new account.')}
-                  </span>
-                  <br />
-                  <span>
-                    {t('Please use ')}
-                    <span
-                      className={`${styles.link} link`}
-                      onClick={() => {
-                        window.open(
-                          'https://lisk.com/blog/development/actions-required-upcoming-mainnet-migration#MigrateanunitiliazedAccount',
-                          '_blank',
-                          'rel=noopener noreferrer'
-                        );
-                      }}
-                    >
-                      {t('external services')}
-                    </span>
-                    {t(' to deposit LSK.')}
+              <>
+                <span>
+                  {t('An initial one-time transfer fee will be deducted from the new account.')}
+                </span>
+                <br />
+                <span>
+                  {t('Please use ')}
+                  <span
+                    className={`${styles.link} link`}
+                    onClick={() => {
+                      window.open(
+                        'https://lisk.com/blog/development/actions-required-upcoming-mainnet-migration#MigrateanunitiliazedAccount',
+                        '_blank',
+                        'rel=noopener noreferrer'
+                      );
+                    }}
+                  >
+                    {t('external services')}
                   </span>
-                </>
-              )}
+                  {t(' to deposit LSK.')}
+                </span>
+              </>
             </div>
           </li>
-          <li className={`${styles.step} ${hasEnoughBalance && styles.green}`}>
+          <li className={`${styles.step} ${hasDepositedTokens ? styles.check : styles.green}`}>
             <div>
               {t('Send a reclaim transaction')}
               <br />
               <span>
                 {t(
-                  'Once you have enough tokens on your new account you will be able to send a transaction.'
+                  'Now that you have enough tokens on your new account, please continue to send the reclaim transaction.'
                 )}
               </span>
             </div>
           </li>
         </ul>
       </section>
-      <DialogLink component="reclaimBalance">
-        <PrimaryButton className={styles.button} disabled={!hasEnoughBalance}>
+      <DialogLink component="reclaimBalance" data={{ tokenID: wallet.token?.[0]?.tokenID }}>
+        <PrimaryButton className={styles.button} disabled={!hasDepositedTokens}>
```

Comment:
user may have blance with nonce 0 they should be allowed to be claim their amount 
its looks wrong to me
@ManuGowda what's your openion here?